### ISSUE TITLE
FPS counter broken

### ISSUE THREAD

**Issue by tagcup** [godotengine/godot#13054]
ca19403 broke the FPS counter for both editor and game (showing 600-800 for both whereas in reality it's like a "fast slideshow" running at ~5FPS), and the editor became much more sluggish on Intel Haswell.

It's probably best to revert it before beta.

**Comment by Calinou** [comment]
Reverted in https://github.com/godotengine/godot/commit/d40b1825fc7c21b80a9482ff7e3ee571dc237d17.

**Comment by tagcup** [comment]
This is not fixed, so can you reopen?

The FPS counter shows 90FPS while it actually runs like a slideshow, so clearly, the value reported is way off.

**Comment by tagcup** [comment]
> please reopen if it isn't.

I can only reopen issues I closed myself, I can't reopen the ones you closed.

**Comment by akien-mga** [comment]
Well it *was* fixed since the problematic commit had been reverted, but likely 62d86b158 adds similar issues again.

**Comment by tagcup** [comment]
Yeah. BTW, this only affects the in-editor FPS. "Print FPS" for the game itself seems to be accurate.

**Comment by akien-mga** [comment]
Can it still be reproduced in the current master branch? If so I think we need a proper fix for 3.0, otherwise many new users will be quite puzzled by incorrect FPS shown in the editor.

**Comment by reduz** [comment]
Fixing this requieres using hardware timers. Too much work so it wont be done for 3.9.
